### PR TITLE
doc: Add Wireless coexistence page under Protocols

### PR DESCRIPTION
**Pull request by richabp**
Add Wireless coexistence page under Protocols
Move “BT radio coexistence” guide to "Wi-Fi Coexistence" page

NCSDK-20383 and NCSDK-20385

### CODE REVIEW

**Comment by wbober** [comment]
@richabp Is it possible to replace the full document title when referencing? I think using 'Yes' or 'Supported' would make the table cleaner.

**Comment by richabp** [comment]
> @richabp Is it possible to replace the full document title when referencing? I think using 'Yes' or 'Supported' would make the table cleaner.

As discussed offline, "Supported" will be used.

**Review by krlo — APPROVED** [review]

File: doc/nrf/protocols.rst [hunk 1]
```diff
@@ -20,5 +20,6 @@ They introduce you to concepts that are important to work with the protocol and
    protocols/thread/index
    protocols/zigbee/index
    protocols/wifi/index
+   protocols/coexistence/index
 
 For details on Wi-Fi coexistence with short range protocols (like Bluetooth LE or Thread), see the :ref:`ug_radio_coex` page.
```
Comment (greg-fer) [inline doc/nrf/protocols.rst:25]: Can this line be moved to the new page? One, it mentions coexistence. Two, we should not have any content after the list of subpages.

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/protocols.rst (hunk 1 above)
Comment (richabp) [reply doc/nrf/protocols.rst]: As per the offline discussion, deleted this line with the link to the https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/app_dev/wifi_coex/index.html#ug-radio-coex page, as the matrix in the new Wireless coexistence page includes this link.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -367,4 +367,4 @@ cJSON
 Documentation
 =============
 
-|no_changes_yet_note|
+Added a page on :ref:`ug_wireless_coexistence` in :ref:`protocols`.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:370]:
```suggestion
* Added a page on :ref:`ug_wireless_coexistence` in :ref:`protocols`.
```

**Review by greg-fer — APPROVED** [review]

**Pull request merged**